runtime/arm64.S

```diff
@@ -676,6 +678,8 @@ L(jump_to_caml):
         stp     d10, d11, [sp, 112]
         stp     d12, d13, [sp, 128]
         stp     d14, d15, [sp, 144]
+        CFI_ADJUST(160)
```

Comment:
Why has this been moved? Isn't it the convention that a `CFI_ADJUST` immediately follows the change to `sp`?

Reply:
That shouldn't be included. I'll remove.

It is left over from experimenting with where the CFI directives should appear in the assembly, after noticing that lldb on macOS sometimes gets confused when debugging. @dustanddreams has indicated that they should be after the instruction which causes the changes they refer to. https://github.com/ocaml/ocaml/pull/13033#discussion_r1558896432

Reply:
Which would mean the instruction sequence should be:
```
        stp     x29, x30, [sp, -160]!
        CFI_OFFSET(29, -160)
        CFI_OFFSET(30, -152)
        add     x29, sp, #0
        CFI_ADJUST(160)
```
With the modifying instruction first and then the CFI directives after that?

Reply:
Yes, that would be the most correct position for CFI directives IMHO.

Reply:
Isn't the instruction modifying `sp` the first one? So the sequence should be

```
        stp     x29, x30, [sp, -160]!
        CFI_OFFSET(29, -160)
        CFI_OFFSET(30, -152)
        CFI_ADJUST(160)
        add     x29, sp, #0
```

Reply:
Hmm, looks like I got confused between `x29` (frame) and `sp`. All the arm64 runtime CFI annotations are `sp`-based, so an even simpler form would be
```
        stp     x29, x30, [sp, -160]!
        CFI_ADJUST(160)
        CFI_OFFSET(29, 0)
        CFI_OFFSET(30, 8)
        add     x29, sp, #0
```